dev/com.ibm.ws.logging/src/com/ibm/ws/logging/data/AccessLogData.java

```diff
@@ -50,20 +50,37 @@
                                             LogFieldConstants.SEQUENCE
     };
 
+    public static String[] ACCESS_LOG_NAMES1_1 = {
```

Comment:
I think you're keeping NAMES for logstashCollector-1.0.  I think you need to keep NAMES1_1 for logstashCollector-1.1, and use new getter methods for the jsonField names.